Fix AssertionError in contentView:repositorie_ids(Existnace Tests)

Content View Test case is failing while comparing(preupugrade To postupgrade) string representation of repository ids included in CVs.
The issue is as below:
```
upgrade_tests/test_existance_relations/test_contentviews.py:39: AssertionError
________________________________________________________________________________ test_positive_cvs_by_repository_ids[pre and post] ________________________________________________________________________________

pre = '5, 1, 2, 85, 133', post = '133, 1, 2, 5, 85'

    @pytest.mark.parametrize("pre,post", cvs_repo, ids=pytest_ids(cvs_repo))
    def test_positive_cvs_by_repository_ids(pre, post):
        """Test repository associations of all CVs post upgrade
    
        :id: c8da27df-7d96-44b7-ab2a-d23a56ea2b2b
    
        :expectedresults: Repositories associations of each CV should be retained
            post upgrade
        """
>       assert pre == post
E       assert '5, 1, 2, 85, 133' == '133, 1, 2, 5, 85'
E         - 5, 1, 2, 85, 133
E         + 133, 1, 2, 5, 85

upgrade_tests/test_existance_relations/test_contentviews.py:39: AssertionError
________________________________________________________________________________ test_positive_cvs_by_repository_ids[pre and post] ________________________________________________________________________________

pre = '55, 62, 74, 75, 116', post = '116, 55, 62, 74, 75'

    @pytest.mark.parametrize("pre,post", cvs_repo, ids=pytest_ids(cvs_repo))
    def test_positive_cvs_by_repository_ids(pre, post):
        """Test repository associations of all CVs post upgrade
    
        :id: c8da27df-7d96-44b7-ab2a-d23a56ea2b2b
    
        :expectedresults: Repositories associations of each CV should be retained
            post upgrade
        """
>       assert pre == post
E       assert '55, 62, 74, 75, 116' == '116, 55, 62, 74, 75'
E         - 55, 62, 74, 75, 116
E         + 116, 55, 62, 74, 75
```

--- a/upgrade_tests/test_existance_relations/test_contentviews.py
+++ b/upgrade_tests/test_existance_relations/test_contentviews.py
@@ -36,6 +36,10 @@ def test_positive_cvs_by_repository_ids(pre, post):
     :expectedresults: Repositories associations of each CV should be retained
         post upgrade
     """
+    if ',' in pre:
+        pre = sorted([num.strip() for num in pre.split(',')])
+    if ',' in post:
+        post = sorted([num.strip() for num in post.split(',')])
     assert pre == post
 
 

